pipelines/broad/arrays/imputation/Imputation.wdl

```diff
@@ -0,0 +1,339 @@
+version 1.0
+
+import "../../../../structs/imputation/ImputationStructs.wdl" as structs
+import "../../../../tasks/broad/ImputationTasks.wdl" as tasks
+
+workflow ImputationPipeline {
+	input {
```

Comment:
Not sure if we have a standard, but most of our WDLs are using a 2-space indent.

Reply:
👍